spec/imsc-hrm.html

```diff
@@ -247,9 +247,164 @@ <h2>Terms and Definitions</h2>
+      minute could be met in multiple ways, with different rendering
+      complexities. Contrast two potential approaches. In the first, 5
+      <a>characters</a> are presented for a fraction of a second, followed by 835
+      <a>characters</a> that are then presented for over 59 seconds. This generates a
+      high rendering complexity for the 835 <a>characters</a>, since there is only a
+      brief time available to paint them. In the second, 210 <a>characters</a> must be
+      painted every 15 seconds, giving 15 seconds to prepare for the next
+      presentation. This has a much lower rendering complexity.</p>
+  
+      <p>The HRM achieves a more accurate representation of the complexity of an
+        <a>IMSC Document Instance</a> at any given time by taking into account
+        its past complexity in addition to its instantaneous complexity. The
+        same approach is commonly used in video to limit bitstream complexity,
+        e.g., the Hypothetical Reference Decoder (HRD) specified in
+        [[iso14496-10]].
+      </p>
+    </section>
+  
+    <section>
+      <h3>How does the HRM measure and limit complexity?</h3>
+  
+      <p>The HRM defines a simple model for the rendering of subtitles and captions, and uses the time it takes to
+      render subtitles and captions according to that model as a proxy for the complexity of the subtitles and captions.
+      Rendering includes drawing region backgrounds, rendering and copying text, and decoding and copying images.
+      Complexity is then limited by requiring that the time to render one subtitle or caption is shorter than the time
+      elapsed since the previous subtitle or caption.
+      </p>
+  
+      <p>The simple model requires only a static analysis of the <a>IMSC Document Instance</a>, requires no fetching of
+      external resources and does not require the <a>IMSC Document Instance</a> to be actually rendered. Several
+      simplifying assumptions are made to achieve this. For example, the model assumes that each <a>character</a> is
+      drawn independently, and accounts for that assumption being, in many cases, false, by assigning different render
+      speeds for different scripts. In general the model is not intended to capture the actual time that an
+      implementation takes to render subtitles and captions, but rather scale with it: a document that is twice as
+      complex according to the model would require roughly twice as many resources to actually render. </p>
+    </section>
+  
+    <section>
+      <h3>Where is the HRM used?</h3>
+  
+      <p>The HRM is typically used prior to distribution of the <a>IMSC Document
+      Instance</a> to end-user, as an integral part of authoring and as a
+      quality check before distribution.</p>
+  
+      <p>When the HRM is used, the consequences of an <a>IMSC Document
+      Instance</a> exceeding the HRM limits depends circumstances:</p>
```

Comment:
Another grammatical nit, plus suggest s/circumstances/context:

```suggestion
      Instance</a> exceeding the HRM limits depend on the context:</p>
```